moves/tests.py

```diff
@@ -1,3 +1,81 @@
+        self.assertTrue(bool(uuid))
+        self.client.get('/')
+        self.assertEqual(self.client.session.get('user_uuid'), uuid)
+
+    @override_settings(DEBUG=True)
+    def test_bootstrap_model_on_page_if_model_with_uuid_with_past_six_hours_exists(self):
+        self.client.get('/')
+        uuid = self.client.session.get('user_uuid')
+        self.client.post('/json/spots/', { 'name': 'The Spot' })
+        self.client.post('/json/moves/', {
+            'user':'user',
+            'spot': Spot.objects.first().id,
+            'time': timezone.now() - datetime.timedelta(hours=3)
+        })
+        recent_move = json.loads(self.client.get('/').context['recent_move'])
+        self.assertEqual(uuid, recent_move['uuid'])
+        self.assertEqual(len(recent_move), 5)
+
+    @override_settings(DEBUG=True)
+    def test_users_name_bootrapped_if_model_beyond_six_hours_ago_exists(self):
+        self.client.get('/')
+        self.client.post('/json/spots/', { 'name': 'The Spot' })
+        self.client.post('/json/moves/', {
+            'user':'user',
+            'spot': Spot.objects.first().id,
+            'time': timezone.now() - datetime.timedelta(hours=9)
+        })
+        recent_move = json.loads(self.client.get('/').context['recent_move'])
+        self.assertEqual(len(recent_move), 1)
+
+    def test_empty_model_bootstrapped_if_no_model_matches(self):
+        recent_move = json.loads(self.client.get('/').context['recent_move'])
+        self.assertEqual(len(recent_move), 0)
+
+class MoveViewSetTests(TestCase):
+    @override_settings(DEBUG=True)
+    def test_move_uuid_matches_user_uuid(self):
+        self.client.get('/')
+        uuid = self.client.session.get('user_uuid')
+        self.client.post('/json/spots/', { 'name': 'The Spot' })
+        self.client.post('/json/moves/', {
+            'user':'User',
+            'spot': Spot.objects.first().id,
+            'time': timezone.now()
+        })
+        move = json.loads(self.client.get('/json/moves/').content)['results'][0]
+        self.assertEqual(move['uuid'], uuid)
+
+    @override_settings(DEBUG=True)
+    def test_moves_index_route_only_displays_moves_from_the_last_six_hours(self):
+        spot = Spot.objects.create(name='The Spot')
+        time_changes = [-10,-9,-8,-7,-6,-5,-4,-3,-2,-1,0,1,2,3,4,5,6,7,8,9,10]
+        for el in time_changes:
+            time = timezone.now() + datetime.timedelta(hours=el)
+            username = 'user' + str(el)
+            Move.objects.create(user=username, spot=spot, time=time)
+        self.client.get('/')
+        moves = json.loads(self.client.get('/json/moves/').content)['results']
+        for move in moves:
+            move_time = datetime.datetime.strptime(move['time'], "%Y-%m-%dT%H:%M:%S.%fZ")
+            if move_time < datetime.datetime.now():
+                difference = datetime.datetime.now() - move_time
+                self.assertFalse(difference.seconds/3600 > 6)
```

Comment:
instead of converting `difference` into seconds and then dividing it by a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) to get hours, I think it would be clearer to just make the comparison between timedeltas. Furthermore, it's generally easier to follow when you assert a condition is true, so how about:

`self.assertTrue(difference < datetime.timedelta(hours=6))`